content/posts/akri-a-year-later.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Akri a Year Later"
+description: "Akri turns One and enters the CNCF Sandbox"
+date: 2021-10-20 00:00:00 +0000 UTC
+authorname: "Kate Goldenring"
+author: "@kate-goldenring"
+tags: ["akri", "cncf"]
+---
+
+A year ago, [we announced Akri](https://cloudblogs.microsoft.com/opensource/2020/10/20/announcing-akri-open-source-project-building-connected-edge-kubernetes/), a Kubernetes operator that discovers and exposes IoT devices to clusters. With hard work and a sprinkle of luck over this past year, we were officially accepted into the [Cloud Native Computing Foundation (CNCF) ](https://www.cncf.io/)as a [Sandbox project](https://www.cncf.io/sandbox-projects/)! This milestone marks an opportunity to look back at the motivation behind Akri and forward at what the future holds for Akri.
```

Comment:
[Cloud Native Computing Foundation (CNC**F) ](h**ttps://www.cncf**.io/)as** -> [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/) as

Reply:
also maybe "with hard work and community support"?